docs/asciidoc/static/deploying.asciidoc

```diff
@@ -0,0 +1,145 @@
+[[deploying-and-scaling]]
+== Deploying and Scaling Logstash
+
+The requirements for your Logstash installation change according to the scale demanded by your use case. In this 
+section, we will examine a range of Logstash installations in increasing order of complexity, starting from a minimal 
+viable installation and adding elements to the system as we go. The example deployments in this section write to an 
+Elasticsearch cluster, but Logstash can write to a large variety of {logstash}output-plugins.html[endpoints].
+
+// [float]
+[[deploying-minimal-install]]
+=== Getting Started: Minimal Installation
```

Comment:
We keep ending up with a lot of "Getting Started" topics. Is it really necessary here?